src/main/java/edu/wpi/grip/ui/Main.java

```diff
@@ -43,18 +42,22 @@ public void start(Stage stage) {
         stage.show();
     }
 
+    private void triggerUnexpectedThrowableEvent(UnexpectedThrowableEvent event) {
```

Comment:
Because I can't access the `eventBus` from within a lambda being created inside of a constructor for a field that hasn't been assigned yet.